src/controllers/connection/index.ts

```diff
@@ -54,14 +54,20 @@ export class ConnectionController extends HTMLController {
    */
   @SuccessResponse(200)
   @Get('/{connectionId}/pin-submission')
-  public async renderPinCode(@Path() connectionId: UUID, @Query() pin?: PIN_CODE | string): Promise<HTML> {
-    this.logger.debug('PIN_SUBMISSION GET: %o', { connectionId, pin })
+  public async renderPinCode(
+    @Request() req: express.Request,
+    @Path() connectionId: UUID,
+    @Query() pin?: PIN_CODE | string
+  ): Promise<HTML> {
+    req.log.trace('pin submission form', { connectionId, pin })
```

Comment:
is this log not covered functionally by the pino-http log? Can you look for this sort of thing everywhere please

Reply:
it does, but logs only req/res and base properties so it's kinda covered but gets fired after the request (removed res object to reduce the load)

```
{"level":20,"time":1726217176493,"pid":76185,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"f506ad0c-5043-4deb-b44a-1b54c3d7d01c","msg":"returning connections page {\"connections\":[{\"company_name\":\"CAY’S CARS LTD\",\"company_number\":\"11111327\",\"status\":\"pending\",\"agent_connection_id\":null,\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"52ed9030-0ed9-4991-b55a-4f0107272ff8\",\"created_at\":\"2024-09-13T07:48:53.415Z\",\"updated_at\":\"2024-09-13T07:48:53.415Z\"},{\"company_name\":\"ZEVENTZUK LIMITED\",\"company_number\":\"11111188\",\"status\":\"verified_both\",\"agent_connection_id\":\"3a3bf137-93bf-4e25-b8b0-78d71d227a56\",\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"1681347b-83f0-47fe-91ad-0623a0c6bdbf\",\"created_at\":\"2024-09-11T08:46:07.940Z\",\"updated_at\":\"2024-09-11T08:48:37.548Z\"},{\"company_name\":\"TECHDIGITAL LIMITED\",\"company_number\":\"11111114\",\"status\":\"pending\",\"agent_connection_id\":null,\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"c6492392-50ea-4551-994a-6520ef2a05bc\",\"created_at\":\"2024-09-10T17:48:29.099Z\",\"updated_at\":\"2024-09-10T17:48:29.099Z\"},{\"company_name\":\"DIGITAL CATAPULT\",\"company_number\":\"07964699\",\"status\":\"unverified\",\"agent_connection_id\":\"2ed1b353-fea7-4978-a31b-0c3129aa372f\",\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"ed92750e-e158-4960-9cea-3290129f6f5e\",\"created_at\":\"2024-09-04T11:32:52.718Z\",\"updated_at\":\"2024-09-06T07:31:17.219Z\"}],\"search\":\"\"}"}
{"level":30,"time":1726217176495,"pid":76185,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"f506ad0c-5043-4deb-b44a-1b54c3d7d01c","req":{"method":"GET","url":"/connection?search=","query":{"search":""},"params":{},"remoteAddress":"::1","remotePort":57496,"headers":{}},"res":{"statusCode":200},"responseTime":8,"msg":"request completed"}
```

Reply:
At the very least I think we should be consistent. Looking around these initial logs happen at different levels and sometimes not at all.

Reply:
so all pino request will be under "info" level 30 but yea, I observed as well, let me refine a little